zellij-server/src/wasm_vm.rs

```diff
@@ -248,6 +256,23 @@ fn host_set_timeout(plugin_env: &PluginEnv, secs: f64) {
     });
 }
 
+fn host_exec_cmd(plugin_env: &PluginEnv) {
+    let mut cmdline: Vec<String> = wasi_read_object(&plugin_env.wasi_env);
+    let command = cmdline.remove(0);
+
+    // Bail out if we're forbidden to run command
+    if plugin_env._allow_exec_host_cmd {
```

Comment:
Almost perfect, but this line needs to be `if !plugin_env._allow_exec_host_cmd {` with the `!`. Right now everything that shouldn't have permissions does and vice versa

Reply:
Ah, sorry, my bad...